research/2022-08-20-introducing-waku-product.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+name:  "Waku for All DApps"
+title:  "Waku for All DApps"
+date:   2022-11-01 00:00:00 +0000
+author: franck
+published: true
+permalink: /waku-for-all-dapps
+categories: waku, dapp, platform, operator
+summary: Waku adoption push has shifted from Web to all platforms. 
+image: /img/black-waku-logo-with-name.png
+discuss: TODO
+---
+
+<!--
+  Audience: Reader curious about Waku/following Waku
+  Goal:
+    - Trigger Ha-ha moment for Waku possibilities
+      - Explain shift from Web to all dApps
+    - Make it clear that Waku is being field tested, it is more than a research project
+    - Ideas on how to contribute/participate/play with Waku
+-->
+
+# Waku for the Web
+
+We are building Waku to be the communication layer for Web3:
+a network and suite of protocols to enable secure, censorship-resistant, privacy-preserving communications that can run on any device.
+
+This involves research efforts in various domains:
+conversational security, protocol incentivization, zero-knowledge, anonymity to name a few.
+
+To ensure we reach our goals, we knew that early dogfooding of Waku was necessary, even if research was still _in progress_ [[1]](#footnote1).
+Thus, we started to push for the adoption of Waku by projects other than the Status app starting 2021.
+
+Waku is the communication component of the Web3 trifecta,
+which originally was Ethereum (contracts), Swarm (storage) and Whisper (communication).
+Hence, it made sense to first target dApps which already uses one of the Web3 pillars: Ethereum. 
+
+As most dApps are web apps,
+we naturally started the development of [js-waku for the browser](https://vac.dev/presenting-js-waku).
+
+Once ready,
+we reached out to dApps to integrate Waku,
+added [prizes to hackathons](https://twitter.com/waku_org/status/1451400128791605254?s=20&t=Zhc0BEz6RVLkE_SeE6UyFA)
+and gave [talks](https://docs.wakuconnect.dev/docs/presentations/).
+
+We also assumed that we would see patterns in the usage of Waku,
+that we would facilitate with the help of [SDKs](https://github.com/status-im/wakuconnect-vote-poll-sdk).
+
+Finally, we created a number of web apps:
+[examples](https://docs.wakuconnect.dev/docs/examples/)
+and [PoCs](https://github.com/status-iM/gnosis-safe-waku).
+
+By discussing with Waku users and watching it being adopted, we learned a few facts:
+
+1. The potential use cases for Waku are varied and numerous,
+2. Many projects are interested in having an embedded decentralized chat feature in their dApp,
+3. There are a number of complex applications, which are not a simple web app, that need Waku as a solution.
+
+(1) means that it is not that easy to create SDKs that facilitate using js-waku's usage.
+
+(2) was a clear candidate for an SDK.
+However, building a chat app is a complex task so the effort was done by the Status app team in the form of [Status Web](https://github.com/status-im/status-web/).
+
+Finally, (3) was the most important lesson.
```

Comment:
Maybe mention somewhere (not sure where) about what we learned in terms of separating out node operators, platforms and developers? From business workshop